src/server/aktørregisterProxy.js

```diff
@@ -0,0 +1,30 @@
+const proxy = require('http-proxy-middleware');
+
+// Mock backend ligger på port 8081
+const aktørregisterUrl = process.env.AKTORREGISTER_URL || 'http://localhost:8081';
+
+const proxyConfig = {
+    changeOrigin: true,
+    target: aktørregisterUrl,
+    // ex fra https://arbeidsgiver.adeo.no/finn-kandidat/aktoerregister/api/v1/identer?identgruppe=AktoerId&gjeldende=true
+    //    til https://app.adeo.no/aktoerregister/api/v1/identer?identgruppe=AktoerId&gjeldende=true
+    // TODO: Fiks URL navn i nais filer
+    pathRewrite: {
+        '^/finn-kandidat': '',
+    },
+    secure: true,
+    xfwd: true,
+    logLevel: 'debug',
+    // TODO: Fjern logging
+    onProxyReq: function onProxyReq(proxyReq, req, res) {
+        // Log outbound request to remote target
+        console.log('-->  ', req.method, req.path, '->', proxyReq.baseUrl + proxyReq.path);
```

Comment:
Hvorfor er dette en TODO, kan vi ikke bare fjerne det nå? Eller bruker vi det til noe?

Reply:
Var som en todo for meg selv så jeg skulle huske å fjerne det :)